### PR TITLE
Added ability to change template of NewJournal

### PR DESCRIPTION
**Pull request by crypdick**
Followed these instructions:
https://groups.google.com/forum/#!topic/TiddlyWiki/ZZOGj6kfqpA

Note: I did not know how to make a blank file at `$:/config/NewJournal/Text`...

### CODE REVIEW

**Comment by Jermolene** [comment]
Hi @crypdick thanks, looks good. Please could you sign the CLA, as described in contributing.md? Many thanks!

**Comment by crypdick** [comment]
@Jermolene done :)

**Comment by Marxsal** [comment]
To make $:/config/NewJournal/Text, just create a new tiddler and change its title to $:/config/NewJournal/Text. When looking for it later, just put "NewJournal" into the advanced search engine and find it under the "System" tab. Or cheat like I do and paste $:/config/NewJournal/Text into a temporary new tiddler and then click on the link in the preview section.

**Comment by Jermolene** [comment]
Thanks @crypdick

**Comment by crypdick** [comment]
Glad to help! I'm happy with my new journal ^_^ 

Question: Now that I've edited my shadow tiddler `$:/core/ui/Buttons/new-journal`, do I have to do something special when I update TW5 to the latest release? I made sure to paste in your version at [83497c1](https://github.com/Jermolene/TiddlyWiki5/commit/83497c13d90a91a3f07122da37f92b5e65a0a453)

File: core/ui/PageControls/new-journal.tid [hunk 1]
```diff
@@ -16,5 +17,7 @@ description: {{$:/language/Buttons/NewJournal/Hint}}
 \end
 <$set name="journalTitleTemplate" value={{$:/config/NewJournal/Title}}>
 <$set name="journalTags" value={{$:/config/NewJournal/Tags}}>
+<$set name="journalText" value={{$:/config/NewJournal/Text}}>
+
 <<journalButton>>
-</$set></$set>
\ No newline at end of file
+</$set></$set>
```
Comment (danielo515) [inline core/ui/PageControls/new-journal.tid]: You are opening three `<$set>` tags, but you are only closing two. Mistake?

File: core/ui/PageControls/new-journal.tid [hunk 2]
```diff
@@ -16,5 +17,7 @@ description: {{$:/language/Buttons/NewJournal/Hint}}
 \end
 <$set name="journalTitleTemplate" value={{$:/config/NewJournal/Title}}>
 <$set name="journalTags" value={{$:/config/NewJournal/Tags}}>
+<$set name="journalText" value={{$:/config/NewJournal/Text}}>
```
Comment (danielo515) [inline core/ui/PageControls/new-journal.tid]: Where is the content of such field configured? I miss an interface on the control panel,where you can set the tags and the title.

Comment (Marxsal) [inline core/ui/PageControls/new-journal.tid]: I think Jerome has already added a configuration panel to the master branch for the text field.

File: core/ui/PageControls/new-journal.tid (hunk 1 above)
Comment (Jermolene) [inline core/ui/PageControls/new-journal.tid]: @danielo515 see my commits 83497c1 and c25a447

Comment (danielo515) [inline core/ui/PageControls/new-journal.tid]: Ok, everything seems fine

**Pull request merged**